ai16z/eliza
# replace ts-node with tsx on @ai16z/agent - fix #288

replace ts-node with tsx on @ai16z/agent



# Relates to:

[Issue #288](https://github.com/ai16z/eliza/issues/288)



# Risks

Low – This change impacts only the TypeScript execution environment by replacing `ts-node` with `tsx`, improving compatibility without affecting core functionality.

# Background

## What does this PR do?

- Replaces `ts-node` with `tsx` to provide better support for handling `.ts` files within the Node.js environment.

## What kind of change is this?

- Bug fixes (non-breaking change that fixes an issue).



## Why are we doing this? Any context or related work?

Replacing `ts-node` with `tsx` resolves an issue with `pnpm start` failing due to `.ts` file handling issues in this environment setup.

# Documentation changes needed?

- My changes require a change to the project documentation.
- I have updated the documentation accordingly.



# Testing

## Where should a reviewer start?

Start by verifying the setup instructions and running `pnpm start` to confirm the fix.

## Detailed testing steps

1. Install dependencies with `pnpm i`.
2. Run `pnpm start`.
3. Confirm that the application starts without errors.
